optashift-employee-rostering-gwtui/src/main/resources/org/optaplanner/openshift/employeerostering/gwtui/public/kie.less

```diff
@@ -3,13 +3,12 @@
   display: flex;
   justify-content: space-between;
   flex-flow: column nowrap;
-  height: 100vh;
-  max-height: 100vh;
+  width: fit-content;
 }
 
 .kie-page__content {
   flex: 1 1 auto;
-  overflow-y: scroll;
+  overflow: visible;
```

Comment:
This might have some undesired effects.. Do you remember why did you have to change that?

Reply:
"There can only be one scrollbar"; If this thing is set to scroll, then we have both the window scrollbars and the page_content scrollbars. The other option would be to set it to "overflow: scroll", but then we have multiple scrollbars

Reply:
Is "hidden" an option?

Reply:
"hidden" would cut off content.